bin/demokit-cli.js

```diff
@@ -19,7 +19,7 @@ if (process.argv[2] === "new")
 }
 else
 {
-    var electronPath = path.join(__dirname, "..", "electron", "node_modules", "electron");
+    var electronPath = path.join(__dirname, "..", "electron", "node_modules", "electron", "dist");
```

Comment:
👍

Reply:
So, I'm having to do `    var electronExecutablePath = path.join(electronPath, fs.readFileSync(path.join(path.dirname(electronPath), "path.txt"), "utf-8"));
` to compensate for this, did you not need this? (specifically, changing to `path.dirname(electronPath)`)

Reply:
Hmm I think I needed something like that?